File: android/src/main/java/com/pumpkin/sud/flutter_sud/GlobalData.java
package com.pumpkin.sud.flutter_sud;

import io.flutter.plugin.common.EventChannel;
import io.flutter.plugin.common.MethodChannel;

public class GlobalData {

    static boolean gameIsTestEnv = false;
    static String appId;
    static String appKey;
    static String userId;
    static String languageCode = "en-US";
    static EventChannel.EventSink eventSink;
    static SudGameViewModel sudGameViewModel;;
    static MethodChannel methodChannel;
}

File: ios/Classes/GlobalData.swift
//
//  GlobalData.swift
//  flutter_sud
//
//  Created by limf on 2024/10/15.
//

import Flutter
import Foundation

class GlobalData {
    static var gameIsTestEnv: Bool = false
    static var appId: String?
    static var appKey: String?
    static var userId: String?
    static var languageCode: String = "en-US"
    static var eventSink: FlutterEventSink?
    static var sudGameEventHandler: SudGameEventHandler?
    static var methodChannel: FlutterMethodChannel?
}
